typescript/introduction/intro-typescript/why-typescript.md

```diff
@@ -0,0 +1,52 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Why TypeScript?
+
+---
+## Content
+
+The main feature, as its name suggests, is that it allows the use of *static types*. 
+
+The static in this context means that types are checked by looking at the written code *without needing to run it*. 
+
+In other words, TypeScript checks your code by making sure you use the same names and shapes where you said you would.
+
+You can also think of it as a **spellchecker for JavaScript**.
+
+This is useful because making sure that the same types of data are used throughout the code can help avoid bugs. It makes the code consistent, and less prone to typos.
+
+It also makes the development faster because you can catch problems early on. 
+
+Since the code doesn't need to run to be type-checked (remember, static types), your editor can integrate with TypeScript and underline in red any type mismatches in the code as you're writing it. This means you write safer code from the start.
+
```

Comment:
We can add an image here:

![ts-safer-code](https://img.enkipro.com/1dca4c660ac1affdf9e903db249599ef.png)

```suggestion
Since the code doesn't need to run to be type-checked (remember, static types), your editor can integrate with TypeScript and underline in red any type mismatches in the code as you're writing it. This means you write safer code from the start.

![ts-safer-code](https://img.enkipro.com/1dca4c660ac1affdf9e903db249599ef.png)
```